Header header
bool is_updated
float64 acc_x   # m/s2
float64 acc_y   # m/s2
float64 yaw_z   # rad/s
